x-pack/test/plugin_api_integration/test_suites/task_manager/task_management.ts

```diff
@@ -247,6 +247,71 @@ export default function ({ getService }: FtrProviderContext) {
         .send({ event: taskId, data })
         .expect(200);
     }
+    it('should schedule a task with rrule', async () => {
+      const dailyTask = await scheduleTask({
+        id: 'sample-recurring-task-id',
+        taskType: 'sampleRecurringTask',
+        schedule: { rrule: { freq: 3, tzid: 'UTC', interval: 1 } },
```

Comment:
Feels like we will want to export `Frequency` and probably `Weekday` from the RRule package, from the TM plugin, to make this easier to folks to use.

Reply:
Done